src/main/java/com/example/TravelAgency/domain/trips/trip/Trip.java

```diff
@@ -0,0 +1,31 @@
+package com.example.TravelAgency.domain.trips.trip;
+
+import com.example.TravelAgency.domain.base.BaseEntity;
+import com.example.TravelAgency.domain.trips.Continent;
+import com.example.TravelAgency.domain.trips.city.City;
+import com.example.TravelAgency.domain.trips.country.Country;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.OneToOne;
+import java.time.LocalDate;
+
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Trip extends BaseEntity {
+
+
+    private String tripName;
+    private Continent continent;
```

Comment:
A mówiłem, @Enumerated(EnumType.STRING)

Reply:
Po za tym chyba nie potrzeba nam kontynentu na wycieczce skoro jest na kraju

Reply:
Jeśli usunę pole kontynent z wycieczki, to nadal będzie można wyszukać wycieczkę tylko po kontynencie ? W sumie podobna sytuacja jest z polem kraj, jest na  wycieczce i  na mieście, to w takim wypadku tu tez coś zmienić ?

Reply:
było by sensowniej pewnie, wyszukać raczej się da bo pole kontynent bedzie w encji kraj która bedzie w encji miasta.
Aczkolwiek po napisaniu tego stwierdzam że jak chcesz to zostaw dla łatwiejszego i szybszego wyszukiwania. Zostawiam Cię z tą decyzją.